tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_battery_power_save_mode.py

```diff
@@ -38,6 +38,6 @@ def test_settings_battery_power_save_mode(self):
         self.assertEqual(self.data_layer.get_setting('powersave.enabled'), False)
 
         battery_settings.tap_turn_on_auto()
-        battery_settings.select('never')
+        battery_settings.select('Never')
```

Comment:
I'd use `data-l10n-id="powerSave-never"` instead. So next time somebody changes the typo, we won't be broken.